srcpkgs/rust/template

```diff
@@ -152,7 +141,7 @@ do_configure() {
 		--prefix=/usr
 		--host=${RUST_TARGET}
 		--target=${RUST_TARGET}
-		--build=${_build_triplet}
+		--build=${XBPS_TRIPLET}
```

Comment:
You sure rust uses the same triplets for ppc? This I can not review, just flag as possibly incorrect

Reply:
See my comment below.